client/packages/common/src/ui/components/domain/Dashboard/StatsPanel.tsx

```diff
@@ -101,12 +101,6 @@ export const StatsPanel: FC<StatsPanelProps> = ({
               ))}
             </Grid>
           )}
-          <Grid item>
-            <BarChartIcon
-              sx={{ height: '50px', width: '125px' }}
-              color="secondary"
-            />
-          </Grid>
```

Comment:
Yeah, good call. I actually thought this was a live chart (guess I hadn't looked very hard 😄), not just a static icon -- so definitely good to remove. :)